src/main/java/uk/gov/hmcts/reform/prl/controllers/EditAndApproveDraftOrderController.java

```diff
@@ -405,6 +407,10 @@ public void sendEmailNotificationToRecipientsServeOrder(
             if (Yes.equals(caseData.getManageOrders().getMarkedToServeEmailNotification())) {
                 manageOrderEmailService.sendEmailWhenOrderIsServed(authorisation, caseData, caseDataUpdated);
             }
+
+            // Check for Automated Hearing Management
+            AutomatedHearingUtils.automatedHearingManagementRequest(authorisation, caseData, caseDataUpdated, manageOrderService);
```

Comment:
there has to be some condition check before we call this method.

Reply:
Removed this automated hearing management call and updated in handleEditAndApproveSubmitted method.